docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM nvidia/cuda:9.2-devel-ubuntu18.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        cuda-libraries-$CUDA_PKG_VERSION qt5-default cmake python3-pyqt5 \
+        libboost-all-dev git wget python3-numpy && apt-get clean
+
+ENV RDBASE=/rdkit-Release_2019_03_1
+RUN wget https://github.com/rdkit/rdkit/archive/Release_2019_03_1.tar.gz && \
+    tar zxvf Release_2019_03_1.tar.gz && cd rdkit-Release_2019_03_1 && \
+    mkdir bld && cd bld && cmake -DRDK_INSTALL_INTREE=OFF \
+    -DCMAKE_INSTALL_PREFIX=/usr -DPYTHON_EXECUTABLE=/usr/bin/python3 .. && \
```

Comment:
you could make the build significantly faster by adding `-DRDK_INSTALL_STATIC_LIBS=OFF -DRDK_BUILD_CPP_TESTS=OFF`

Reply:
Done